### DISCUSSION TITLE
【萌新求教】好友消息不能生效而群聊消息可以

### DISCUSSION THREAD

**Discussion by Nuit5566** [nonebot/discussions#193] [问答] [answered]
如题，接收到好友消息时没有反应，同时终端提示 Cannot guess message type to reply! 但在群聊里指令可以正常触发
![image](https://user-images.githubusercontent.com/63411237/184139409-303f334d-b3a3-415b-aa60-0869e486562f.png)
![image](https://user-images.githubusercontent.com/63411237/184139312-0a5f13cf-a2fa-45aa-b256-43455864bd84.png)

**Comment by yanyongyu** [comment, marked as answer]
这是个onebot适配器新版本的bug，参考nonebot/adapter-onebot#24

**Reply by Nuit5566** [reply]
感谢，解决了
